docs/db/graphql.md

```diff
@@ -0,0 +1,217 @@
+# Examples of graphql (queries)
+
+> Note: that all queries present here can be ran in your own school. For this you must login with an admin role and access the following route <https://((DOMAIN))/graphiql>. Depending on our role the tables permissions may differ.
+
+## Simple queries to get info
+
+- The following query gives the basic information of a given user :
+
+```graphql
+query getUserInfo($name: String!) {
+  user(where: {login: {_eq: $name}}) {
+    profile
+    login
+    attrs
+    auditRatio
+    campus
+    createdAt
+    totalDown
+    totalUp
+    updatedAt
+  }
+}
+```
+
+Query variable:
+
+```graphql
+{"name": "Joao"}
+```
+
+---
+
+- The following query gives the record of a given user, the output would be :
+  - author of the record
+  - the time that the record was created and will end
+  - message/reason for the ban
+
+```graphql
+query bannedUsers($name: String!) {
+  record(where: {user: {login: {_eq: $name}}}) {
+    authorLogin
+    banEndAt
+    createdAt
+    message
+  }
+}
+```
+
+Query variable:
+
+```graphql
+{"name": "Joao"}
+```
+
+---
+
+- The following query gives a list of groups given the project `name`. The output will be the `captainLogin` and the `userLogin` from all the members of that group.
+
+```graphql
+query getGroupInfo($object: String!) {
+  group(where: {object: {name: {_eq: $object}}}) {
+    captainLogin
+    members {
+      userLogin
+    }
+  }
```

Comment:
maybe you can add the status of the group too, to see which group is doing what on the project (and the eventId?)